sdk/textanalytics/ai-text-analytics/CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 # Release History
 
-## 1.0.0-preview.5 (Unreleased)
+## 1.0.0-preview.5 (2020-05-26)
+- [Breaking] Renamed all result array types that extend JavaScript's base `Array` class to end with the word `Array` instead of `Collection` (e.g. `AnalyzeSentimentResultCollection` is now `AnalyzeSentimentResultArray`)
+- [Breaking] Renamed `score` to `confidenceScore` in both the `Match` and `Entity` types.
+- [Breaking] Removed the `graphemeOffset` and `graphemeLength` properties of both the `Match` and `Entity` types.
+- [Breaking] Renamed the `graphemeCount` property of `TextDocumentStatistics` back to `characterCount`
+- Added a `text` property to `SentenceSentiment` that contains the sentence text
+- [Breaking] Removed the `warnings` property of `SentenceSentiment`.
+- Added `warnings` property to every document response object containing a list of `TextAnalyticsWarning` objects relevant to the corresponding document.
+- Migrated to using the Text Analytics v3.0 (General Availability) service endpoint.
```

Comment:
could we add to the TA Readme file that our library targets 3.0 service version?

Reply:
Sure, I'll add it to this PR: https://github.com/Azure/azure-sdk-for-js/pull/8928